docs/source/docs/cli.md

```diff
@@ -40,6 +40,16 @@ configuration can be specified in the configuration file. Please see
 
 ### Optional arguments
 
+- **CPU allocation**:
```

Comment:
What happens if users enter 0, or a number higher than the CPU cores they have? Do we need to check for this if this behaviour causes issues?

Reply:
I only limit the CPU if this number if > 0, thus I will add an error